samples/ResponseCachingApp/test-run.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
It might be worth trying to use an `.fsx` file since not everyone has bash.(yes I know the repo has a bunch of `ps1` files which I would make the same argument for but was done before)

Reply:
Make sense, I'll write an `.fsx` file soon.

Reply:
Done.